modules/end-user-guide/partials/proc_adding-vs-code-extensions.adoc

```diff
@@ -0,0 +1,59 @@
+
+[id="adding-vs-code-extensions"]
+= Adding `VS Code` extensions
+
+To have the Microsoft Visual Studio Code - Open Source IDE automatically install chosen extensions, you have to add an `extensions.json` file to the remote Git repository that contains your project source code and that will be cloned into workspaces.
+
+.Prerequisites
+* {prod-short} has network access to the link:https://www.open-vsx.org/[OpenVSX registry].
++
+[TIP]
+====
+To install recommended extensions in a restricted environment, consider these options:
+
+* xref:administration-guide:configuring-the-open-vsx-registry-url.adoc[] to point to your OpenVSX registry.
+
+* xref:defining-a-common-ide.adoc[].
+
+* link:https://code.visualstudio.com/docs/editor/extension-marketplace#_install-from-a-vsix[Installing extensions from VSX files].
+====
+
+.Procedure
+
+. Get the publisher and extension names of each chosen extension:
+
+.. Find the extension on the link:https://www.open-vsx.org/[Open VSX registry website] and open the extension's listing page.
+
+.. Right-click btn:[DOWNLOAD] and copy its URL.
```

Comment:
This sub-procedure for determining the 'publisher' and the 'extension' is overly complicated and too brittle. All the right and left-clicking, but only if something or other...

Just use the URL in the store, which contains all the needed info:

E.g.:

* https://www.open-vsx.org/extension/redhat/java
* https://www.open-vsx.org/extension/rust-lang/rust-analyzer
* https://www.open-vsx.org/extension/GitLab/gitlab-workflow

=> https://www.open-vsx.org/extension/&lt;publisher&gt;/&lt;extension&gt;/

Reply:
Oh, thank you so much! I'm glad you've looked at it!

Reply:
Here's what I did here:

```
. Get the publisher and extension names of each chosen extension:

.. Find the extension on the link:https://www.open-vsx.org/[Open VSX registry website] and copy the URL of the extension's listing page.

.. Extract the __<publisher>__ and __<extension>__ names from the copied URL:
+
[subs="+quotes"]
----
https://www.open-vsx.org/extension/__<publisher>__/__<extension>__
----
```